src/awkward/_v2/contents/bitmaskedarray.py

```diff
@@ -354,7 +354,7 @@ def mergeable(self, other, mergebool):
         if isinstance(
             other,
             (
-                ak._v2.contents.emptyArray.EmptyArray,
+                ak._v2.contents.emptyarray.EmptyArray,
```

Comment:
or should it just be `ak._v2.contents.emptyArray` ? Contents module already imports all the things. In this module, there are a number of these, but the others follow the lower-case spelling.

Reply:
There shouldn't be any lower camelCase module names at all.

Using

```bash
egrep -r '\b[a-z]+[A-Z]' src/awkward/_v2/**/*.py | egrep -v '\bto' | egrep -v '\bkMax' | egrep -v '\bkSlice'
```

I found the one you just found and another one in indexedarray.py (in addition to [the original](https://github.com/scikit-hep/awkward-1.0/pull/1172/files)), though that's it; the rest of the regex matches are not instances of this copy-paste error.

```
src/awkward/_v2/_connect/numexpr.py:# def getArguments(names, local_dict=None, global_dict=None):
src/awkward/_v2/_connect/numexpr.py:#     context = numexpr.necompiler.getContext(kwargs, frame_depth=1)
src/awkward/_v2/_connect/numexpr.py:#         numexpr.necompiler._names_cache[expr_key] = numexpr.necompiler.getExprNames(
src/awkward/_v2/_connect/numexpr.py:#     arguments = getArguments(names, local_dict, global_dict)
src/awkward/_v2/_connect/numexpr.py:#     arguments = getArguments(names, local_dict)
src/awkward/_v2/_connect/numpy.py:#         rowsA = len(A)
src/awkward/_v2/_connect/numpy.py:#         colsA = first
src/awkward/_v2/_connect/numpy.py:#         rowsB = len(B)
src/awkward/_v2/_connect/numpy.py:#         colsB = first
src/awkward/_v2/_connect/numpy.py:#         if colsA != rowsB:
src/awkward/_v2/contents/bitmaskedarray.py:                ak._v2.contents.emptyArray.EmptyArray,
src/awkward/_v2/contents/indexedarray.py:                ak._v2.contents.emptyArray.EmptyArray,
src/awkward/_v2/contents/listarray.py:            next_content = asListOffsetArray64._content
src/awkward/_v2/highlevel.py:_dir_pattern = re.compile(r"^[a-zA-Z_]\w*$")
src/awkward/_v2/operations/convert/ak_from_json_schema.py:        "`\\uXXXX`" escape sequences. If an `"enum"` is given, strings are represented
src/awkward/_v2/operations/convert/ak_from_json_schema.py:        if schema.get("minItems") == schema.get("maxItems") != None:  # noqa: E711
src/awkward/_v2/operations/convert/ak_from_json_schema.py:            assert ak._v2._util.isint(schema.get("minItems"))
src/awkward/_v2/operations/convert/ak_from_json_schema.py:            instructions.append(["FixedLengthList", schema.get("minItems")])
src/awkward/_v2/operations/convert/ak_from_json_schema.py:            out = ak._v2.forms.RegularForm(content, size=schema.get("minItems"))
src/awkward/_v2/operations/convert/ak_to_buffers.py:    Dask, but partition numbers can be emulated by prepending a fixed `"partN-"`
```

As for whether it should be `ak._v2.contents.emptyarray.EmptyArray` or `ak._v2.contents.EmptyArray`, I slightly prefer the latter, though I'd make it fit the context of the surrounding code, if relevant.

Reply:
OK, in that case my version is fine, right?

Reply:
Your version is fine, though there's another one just like this in indexedarray.py:

```
src/awkward/_v2/contents/indexedarray.py:                ak._v2.contents.emptyArray.EmptyArray,
```

Since you reported the first one in unmaskedarray.py, I figure it's only a matter of time before you hit the one in indexedarray.py, too.

Meanwhile, I'm more worried about the build failing. You made a change that can't affect compilation, and yet every compilation is failing (in pybind11, which was recently updated...).